internal/imgutil/imgutil.go

```diff
@@ -34,7 +33,7 @@ func GetRemoteImage(imgRef string) (v1.Image, error) {
 // ExtractEnvbuilderFromImage reads the image located at imgRef and extracts
 // MagicBinaryLocation to destPath.
 func ExtractEnvbuilderFromImage(ctx context.Context, imgRef, destPath string) error {
-	needle := filepath.Clean(constants.MagicBinaryLocation)[1:] // skip leading '/'
+	needle := ".envbuilder/bin/envbuilder"
```

Comment:
Can we use a constant rather for `.envbuilder` in this and `runCacheProbe`?

Reply:
Sure thing -- I was perhaps a bit over-zealous in de-constantizing.